### PR TITLE
Update cluster-install manifests

### PR DESCRIPTION
**Pull request by danilo-gemoli**
This goes together with https://github.com/openshift/ci-tools/pull/4413

### CODE REVIEW

File: clusters/_cluster-install/build10.yaml [hunk 1]
```diff
@@ -6,5 +6,132 @@ onboard:
...
+        name: '^.+\-worker-.+$'
+      inline: [{"op": "add", "path": "/spec/template/spec/providerSpec/value/blockDevices/0/ebs/volumeSize", "value": 800}]
+    - type: json-patch
+      matches:
+      - kind: MachineSet
+        name: '^.+\-infra-.+$'
+      inline: [
+        {"op": "add", "path": "/spec/replicas", "value": 1},
+        {"op": "add", "path": "/spec/template/spec/providerSpec/value/instanceType", "value": "m6g.2xlarge"},
+        {"op": "add", "path": "/spec/template/spec/taints", "value": [{"effect": "NoSchedule","key": "node-role.kubernetes.io/infra","value": "reserved"}]}]
```
Comment (liangxia) [inline clusters/_cluster-install/build10.yaml:38]: yamllint alerts that,
```
clusters/_cluster-install/build10.yaml
  38:95     error    too few spaces after comma  (commas)
  38:134    error    too few spaces after comma  (commas)
```

File: clusters/_cluster-install/build11.yaml [hunk 2]
```diff
@@ -19,3 +19,33 @@ onboard:
...
+        name: '^.+\-worker-.+$'
+      inline: [{"op": "add", "path": "/spec/template/spec/providerSpec/value/blockDevices/0/ebs/volumeSize", "value": 800}]
+    - type: json-patch
+      matches:
+      - kind: MachineSet
+        name: '^.+\-infra-.+$'
+      inline: [
+        {"op": "add", "path": "/spec/replicas", "value": 1},
+        {"op": "add", "path": "/spec/template/spec/providerSpec/value/instanceType", "value": "m6g.2xlarge"},
+        {"op": "add", "path": "/spec/template/spec/taints", "value": [{"effect": "NoSchedule","key": "node-role.kubernetes.io/infra","value": "reserved"}]}]
```
Comment (liangxia) [inline clusters/_cluster-install/build11.yaml:48]: and this line, too
```
clusters/_cluster-install/build11.yaml
  48:95     error    too few spaces after comma  (commas)
  48:134    error    too few spaces after comma  (commas)
```

File: clusters/_cluster-install/build10.yaml (hunk 1 above)
Comment (danilo-gemoli) [reply clusters/_cluster-install/build10.yaml:38]: Nice, thanks!

**Pull request merged**